### ISSUE TITLE
Sauce Service Not Updating Jobs With Needed Data in 6.10.4

### ISSUE THREAD

**Issue by joshmgrant** [webdriverio/webdriverio#6234]
[//]: # NOTE: This repository only maintains packages that are listed in the [Readme](https://github.com/webdriverio/webdriverio/blob/master/README.md#packages). Please make sure that your issue is directly caused by one of these packages and if not file an issue in the correct 3rd party package repository.

**Environment (please complete the following information):**
 - **WebdriverIO version:** 6.10.4
 - **Mode:** WDIO testrunner
 - **If WDIO Testrunner, running sync/async:** sync
 - **Node.js version:** 12.14.1
 - **NPM version:** 6.13.4
 - **Browser name and version:** multiple
 - **Platform name and version:** multiple
 - **Additional wdio packages used (if applicable):** @wdio/sauce-service [6.10.4]

**Config of WebdriverIO**
An example of how you configured WebdriverIO config:

Here is the Sauce-specifc portion:

```js
config.region = 'us';

// ============
// Capabilities
// ============
config.capabilities = [
    /**
     * Desktop browsers
     */
    {
        browserName: 'googlechrome',
        platformName: 'Windows 10',
        browserVersion: 'latest',
        'sauce:options': {
            ...defaultBrowserSauceOptions,
        },
        ...chromeOptions,
    },
    {
        browserName: 'safari',
        platformName: 'iOS',
        deviceName: 'iPhone 11 Simulator',
        deviceOrientation: 'portrait',
        platformVersion: '14.0',
        name: 'testing-iphone'
    },
    {
        browserName: 'chrome',
        platformName: 'android',
        deviceName: 'Google Pixel 3 GoogleAPI Emulator',
        deviceOrientation: 'portrait',
        platformVersion: '11.0',
        name: 'testing-pixel'
    }
];
```

**Describe the bug**
When I execute tests, I do the following:

- Set Sauce username, accesskey, and region as needed, 
- Update WDIO and WDIO/Sauce-Service to 6.10.4
- Run tests with `wdio test/configs/wdio.saucelabs.conf.js`
- Look at test results in Sauce Labs

**To Reproduce**
A minimal example can be found in the `webdriverio/web/webdriver` directory of this branch: 

https://github.com/saucelabs-training/demo-js/tree/wdio_sauce_job_bug

Run `npm install` followed by `npm run test.saucelabs.us`. 

**Expected behavior**
The Sauce Jobs dashboard will have the appropriate test job information updated. The test status will update to pass/fail, and the test job name will not be the default `Untitled job 12345abc`. 

**Log**

Logs can be found in the following Sauce Labs jobs: 

https://app.saucelabs.com/tests/ef4def9663a94c99b3dd38f7b3477ea0#19
https://app.saucelabs.com/tests/c9b796357c6e4601a996e3e6b3e2a56b#21

**Comment by christian-bromann** [comment]
> The Sauce Jobs dashboard will have the appropriate test job information updated. The test status will update to pass/fail, and the test job name will not be the default `Untitled job 12345abc`.

@joshmgrant I was not able to reproduce this:

![Screenshot 2020-12-15 at 11 13 21](https://user-images.githubusercontent.com/731337/102201900-faf8fb00-3ec6-11eb-92e3-e72fba368c2c.png)

Maybe just a temporary Glitch in the API?

**Comment by christian-bromann** [comment]
@joshmgrant this should be fixed by #6245. Please update your dependencies and try again. Happy to re-open in case the bug still appears.
